ext/web/10_filereader.js

```diff
@@ -363,6 +364,8 @@
       // alias for readAsArrayBuffer
       this.#readOperation(blob, { kind: "Text", encoding });
     }
+
+    [SymbolToStringTag] = "FileReader";
```

Comment:
This is weird... there is a `webidl.configurePrototype` already. Maybe because it inherits from EventTarget, which does not use `webidl.configurePrototype` yet?

Reply:
You are right. EventTarget wasn't configured